app/controllers/api/cycles_controller.rb

```diff
@@ -1,17 +1,26 @@
 module Api
   class CyclesController < ApiApplicationController
     before_action :set_curriculum, only: [:index]
+    before_action :set_cycle, only: [:show]
 
     def index
       cycles = @curriculum.cycles
       render json: cycles,
              only: %i[id name learning_goals_description]
     end
 
+    def show
+      render json: @cycle, except: %i[created_at updated_at curriculum_id]
```

Comment:
Suggestion: Could be better to explicitly declare what includes instead of what excludes.

Reply:
done, thanks!